### ISSUE TITLE
Electron app doesn't support Windows Narrator for screen reading

### ISSUE THREAD

**Issue by nishant09** [electron/electron#6907]
* Electron version:0.34
* Operating system: Windows

Hi,
I'm trying to test Accessibility for my electron app(Microsoft Azure Storage Explorer). I found that my app doesn't support Windows Narrator. However, I was able to use NVDA tool. It works perfectly fine.

Also, my app is working fine with Mac OS' VoiceOver utility.

Could someone give me any insight on this ? Does the latest version of Electron support Windows Narrator.

**Comment by zcbenz** [comment]
Can you try latest version? 0.34 is quite old and we had lots of accessibility fixes in later versions.

**Comment by nishant09** [comment]
sure. we are in the process of adopting the latest version.

**Comment by kevinsawicki** [comment]
@nishant09 any updates on whether you still see this issue in the latest Electron version?

**Comment by nishant09** [comment]
Hi.. We haven't adopted the latest version yet. It will another 2 weeks. I should be get back to you then. sorry for the delay. looks like it is not a straightforward process

**Comment by kevinsawicki** [comment]
> It will another 2 weeks.

Closing this out for now, please comment back once you upgrade if this is still an issue and we can reopen if this is still happening in 1.4+

**Comment by alexeykuzmin** [comment]
@nishant09 Did you try to use Narrator with a newer version of Electron?

**Comment by digitalnetbizz2** [comment]
Hello, 

I am using Electron build (below) and running on Windows 10. I can confirm that Electron app doesn't work with Windows 10 built-in Narrator.

"electron": "^1.6.10",

**Comment by alexeykuzmin** [comment]
@digitalnetbizz2 Can you please try the latest stable Electron 1.8.2?
